### PR TITLE
add info-div for listening messages coming from socket

### PR DESCRIPTION
**Pull request by aitchkhan**

### CODE REVIEW

File: client/src/index.html [hunk 1]
```diff
@@ -100,5 +113,28 @@
             </div>
             <div class="control" id="control-status"></div>
         </div>
+
+        <div id="info-div" class="info-div" style="visibility: hidden">
+            <div>
+                <ul>
+                    <li>Attentive:
```
Comment (alyyousuf7) [inline client/src/index.html]: nit: `Attentive` on new line maybe, just like others?

File: client/src/index.js [hunk 2]
```diff
@@ -5,6 +5,19 @@ const CREATE_SESSION_ENDPOINT = '/api/create_session';
 
 window.onload = setupClient;
 
+window.onkeyup = function onKeyUp(params) {
+    // on "i" key press
+    if(event.keyCode === 73) {
+        const infoDiv = document.getElementById('info-div');
+        if(infoDiv.style.visibility  === 'hidden') {
```
Comment (alyyousuf7) [inline client/src/index.js]: nit: Please take care of double spaces before `===`

Comment (aitchkhan) [reply client/src/index.js]: I rely too much on lint :(

File: client/src/index.js [hunk 3]
```diff
@@ -22,6 +35,43 @@ function showStatus(status) {
     showControlbar('status');
 }
 
+function recordEvent(event) {
+    const data = event.data ? JSON.parse(event.data) : null;
+
+    switch (data.type) {
+        case "detection":
```
Comment (alyyousuf7) [inline client/src/index.js]: We've been using single quotes in `js`.

File: client/src/index.js [hunk 4]
```diff
@@ -22,6 +35,43 @@ function showStatus(status) {
...
+            document.getElementById("transcript").innerHTML = JSON.stringify(data.data.transcript);
+            break;
+    }
+}
+
+function resetInfoDiv() {
+    document.getElementById("attentive").innerHTML = 0; 
+    document.getElementById("hand_gesture").innerHTML = 0; 
+    document.getElementById("face_attributes").innerHTML = 0; 
+    document.getElementById("transcript").innerHTML = 0;
```
Comment (alyyousuf7) [inline client/src/index.js]: Oh, setting to `0` empties the HTML? 😮  I didn't know that.

Comment (aitchkhan) [reply client/src/index.js]: No. Just keeping the defaults to 0.
`empty = ""`

File: client/src/index.js (hunk 3 above)
Comment (aitchkhan) [reply client/src/index.js]: my bad :disappointed:

**Pull request merged**